src/main/java/com/autotune/queue/RecMgrQueue.java

```diff
@@ -0,0 +1,72 @@
+/*******************************************************************************
```

Comment:
Same thing here as well

Reply:
AutotuneQueueFactory using both the instances so I have to put it as part of this PR if you want it in separate package then I have create and move this class.
Please let me know your decision?